.gitignore

```diff
@@ -14,4 +14,6 @@ target/
 #  be found at https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore
 #  and can be added to the global gitignore or merged into this file.  For a more nuclear
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
-#.idea/
\ No newline at end of file
+#.idea/
+
+n_plus_one_example/.env
```

Comment:
We can probably change this to just `.env` and then any `.env` file from any dir should be ignored!